### PR TITLE
fix: Fix flaky test shouldResumeExporting in ExportEndpointIT

### LINKED ISSUE #10628: `ExportingEndpointIT.shouldResumeExporting` is flaky
**Summary**

> Try to answer the following as best as possible

- How often does the test fail? Once so far
- Does it block your work? No
- Do we suspect that it is a real failure? Maybe?

**Failures**

> Outline known failure cases, e.g. a failed assertion and its stacktrace obtained from Jenkins

<details><summary>Example assertion failure</summary>
<pre>
Error:    Run 1: ExportingEndpointIT.shouldResumeExporting:103->startProcess:131 ? ClientStatus Command 'CREATE_WITH_AWAITING_RESULT' rejected with code 'NOT_FOUND': Expected to find process definition with process ID 'processId', but none found
</pre>
</details>

**Hypotheses**

No idea, test looks correct and like it couldn't fail in this way.

**Logs**

> If possible, provide more context here, e.g. standard output logs, link to build, etc.

<details><summary>Logs</summary>
<pre>
06:22:25.304 [] [ForkJoinPool-3-worker-1] INFO  io.camunda.zeebe.test.util.SocketUtil - Choosing next port 1232 for test fork 2 with range PortRange{host='localhost', basePort=1225, maxOffset=100, currentOffset=8}
06:22:25.306 [] [ForkJoinPool-3-worker-1] INFO  io.zeebe.containers.cluster.ZeebeCluster - Starting cluster zeebe-cluster with 2 brokers, 2 gateways, 2 partitions, and a replication factor of 1
06:22:25.307 [] [testcontainers-lifecycle-0] INFO  docker[localhost:5000/camunda/zeebe:current-test] - Creating container for image: localhost:5000/camunda/zeebe:current-test
06:22:25.307 [] [testcontainers-lifecycle-2] INFO  docker[localhost:5000/camunda/zeebe:current-test] - Creating container for image: localhost:5000/camunda/zeebe:current-test
06:22:25.402 [] [testcontainers-lifecycle-0] INFO  docker[localhost:5000/camunda/zeebe:current-test] - Container localhost:5000/camunda/zeebe:current-test is starting: 7e57ba67130ef25409bde8ac22a2ef34f013312498770e08eeb67be9bd0969f7
06:22:25.447 [] [testcontainers-lifecycle-2] INFO  docker[localhost:5000/camunda/zeebe:current-test] - Container localhost:5000/camunda/zeebe:current-test is starting: 36383c1e568ea5551e92acd6d1f6a1657b49f9c47d08cea95fa3fdac45e97738
06:22:33.373 [] [ducttape-1] INFO  org.testcontainers.containers.wait.strategy.HttpWaitStrategy - /elastic_greider: Waiting for 120 seconds for URL: http://localhost:49234/ready (where port 49234 maps to container port 9600)
06:22:33.958 [] [ducttape-3] INFO  org.testcontainers.containers.wait.strategy.HttpWaitStrategy - /festive_buck: Waiting for 120 seconds for URL: http://localhost:49238/ready (where port 49238 maps to container port 9600)
06:22:37.756 [] [ducttape-1] INFO  io.zeebe.containers.ZeebeTopologyWaitStrategy - /elastic_greider: Waiting for PT2M for topology to have at least 2 brokers, 2 partitions with 1 replicas, and each partition to have a leader
06:22:37.939 [] [ducttape-3] INFO  io.zeebe.containers.ZeebeTopologyWaitStrategy - /festive_buck: Waiting for PT2M for topology to have at least 2 brokers, 2 partitions with 1 replicas, and each partition to have a leader
06:22:38.889 [] [ducttape-0] TRACE io.zeebe.containers.ZeebeTopologyWaitStrategy - /elastic_greider: Topology: TopologyImpl{brokers=[BrokerInfoImpl{nodeId=1, host='zeebe-broker-1', port=26501, version=8.2.0-SNAPSHOT, partitions=[PartitionInfoImpl{partitionId=2, role=LEADER, health=HEALTHY}]}, BrokerInfoImpl{nodeId=0, host='zeebe-broker-0', port=26501, version=8.2.0-SNAPSHOT, partitions=[PartitionInfoImpl{partitionId=1, role=LEADER, health=HEALTHY}]}], clusterSize=2, partitionsCount=2, replicationFactor=1, gatewayVersion=8.2.0-SNAPSHOT}
06:22:38.891 [] [testcontainers-lifecycle-0] INFO  docker[localhost:5000/camunda/zeebe:current-test] - Container localhost:5000/camunda/zeebe:current-test started in PT13.583746734S
06:22:39.162 [] [ducttape-2] TRACE io.zeebe.containers.ZeebeTopologyWaitStrategy - /festive_buck: Topology: TopologyImpl{brokers=[BrokerInfoImpl{nodeId=0, host='zeebe-broker-0', port=26501, version=8.2.0-SNAPSHOT, partitions=[PartitionInfoImpl{partitionId=1, role=LEADER, health=HEALTHY}]}, BrokerInfoImpl{nodeId=1, host='zeebe-broker-1', port=26501, version=8.2.0-SNAPSHOT, partitions=[PartitionInfoImpl{partitionId=2, role=LEADER, health=HEALTHY}]}], clusterSize=2, partitionsCount=2, replicationFactor=1, gatewayVersion=8.2.0-SNAPSHOT}
06:22:39.164 [] [testcontainers-lifecycle-2] INFO  docker[localhost:5000/camunda/zeebe:current-test] - Container localhost:5000/camunda/zeebe:current-test started in PT13.85725724S
</pre>
</details>

### PR DESCRIPTION
**Pull request by rodrigo-lourenco-lopes**
## Description

This PR fixes the flaky test shouldResumeExporting in ExportEndpointIT.

Since the cluster being used has more than one partition, it might happen that the start process command happens before the remaining partitions have received the process. That is because when a process gets deployed, it will be first sent to partition 1 and then forwarded to the remaining partitions.

Since the test here is only trying to assert that exporting gets resumed, then it is ok to just publish a message that will generate a record instead of deploying a process.
## Related issues

closes https://github.com/camunda/zeebe/issues/10628

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Other teams:
If the change impacts another team an issue has been created for this team, explaining what they need to do to support this change.
- [ ] [Operate](https://github.com/camunda/operate/issues)
- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

**Comment by rodrigo-lourenco-lopes** [comment]
bors merge

File: qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ExportingEndpointIT.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ void shouldResumeExporting() {
     final var actuator = getActuator();
     actuator.pause();
 
-    startProcess();
+    client.newPublishMessageCommand().messageName("Test").correlationKey("1").messageId("1").send();
```
Comment (deepthidevaki) [inline qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ExportingEndpointIT.java:82]:
```suggestion
    client.newPublishMessageCommand().messageName("Test").correlationKey("1").messageId("1").send().join();
```

Wait for the future to make it deterministic.

**Review by deepthidevaki — APPROVED** [review]
Thanks @rodrigo-lourenco-lopes 

:x: Please also update `shouldPauseExporting` the same way. It can also be flaky due to the same reason.

**Comment by rodrigo-lourenco-lopes** [comment]
bors merge

**Comment by rodrigo-lourenco-lopes** [comment]
Backports failed due to merge conflicts, a manual backport was created in https://github.com/camunda/zeebe/pull/14821

**Pull request merged**